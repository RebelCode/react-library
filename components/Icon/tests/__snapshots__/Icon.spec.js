import React from "react";
import { render, cleanup } from "@testing-library/react";
import { Icon } from "../../index";

afterEach(cleanup);

describe("Icon", () => {
  const Comp = (
    <Icon aria-label="Icon"  />
  );

  it("should take a snapshot", () => {
    const { asFragment } = render(Comp);

    expect(asFragment(<Icon />)).toMatchSnapshot();
  });

  it("renders correctly", () => {
    const { queryByLabelText } = render(Comp);

    expect(queryByLabelText("Icon")).toBeTruthy();
  });
});
