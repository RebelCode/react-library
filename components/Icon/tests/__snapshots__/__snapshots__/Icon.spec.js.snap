// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Icon should take a snapshot 1`] = `
<DocumentFragment>
  <svg
    aria-label="Icon"
    fill="currentColor"
    viewBox="0 0 20 20"
  >
    <g />
  </svg>
</DocumentFragment>
`;
